users_ldap_populate/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===================
+users_ldap_populate
+===================
+
+This module extends the functionality of auth_ldap by adding functionality so
+that users can be populated from a LDAP server.
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Install the module users_ldap_populate from the module list.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to 'Settings' -> 'General Settings'
+#. In the LDAP category click on or add a LDAP configuration
+#. In the pop-up enter all the LDAP settings
+#. Click the populate button
+#. A new pop-up will notify you of how many users are added
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/{server-tools}/{10.0}
```

Comment:
The link doesn't work, even if you remove the brackets in the URL.
You can use: `https://runbot.odoo-community.org/runbot/repo/149`